src/components/ChatLog.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { useState } from 'react';
+import './ChatLog.css';
+import ChatEntry from '../components/ChatEntry';
+
+const ChatLog = (props) => {
+    return props.entries.map((message) => {
+      return <ChatEntry incrementLikes={props.incrementLikes} decrementLikes={props.decrementLikes} sender={message.sender} body={message.body} timeStamp={message.timeStamp} key={message.id} />
```

Comment:
Prefer to linewrpa this for legibility
```js
      return <ChatEntry 
        incrementLikes={props.incrementLikes}
        decrementLikes={props.decrementLikes}
        sender={message.sender}
        body={message.body}
        timeStamp={message.timeStamp}
        key={message.id} />
```

Reply:
👀 We should pass along the `id` and `liked` information about each message
```js
      return <ChatEntry 
        ... other attributes...
        id={message.id}
        liked={message.liked}
        key={message.id} />
```

Note that the `key` and `id` are two distinct attributes. `id` isn't a special name. We are simply passing the id of the message record down to the component. `key` _is_ special in React, and is used when we generate a list of components. We will not see `key` appear in the `props` passed to the component.